polynote-kernel/src/main/scala/polynote/kernel/interpreter/python/PythonInterpreter.scala

```diff
@@ -129,6 +131,21 @@ class PythonInterpreter private[python] (
     resState        <- run(compiled, decls, globals, state)
   } yield resState
 
+  private def extractParamsFast(jep: Jep, jediDefinition: PyObject): List[(String, String)] = {
```

Comment:
This function is unrelated... probably should have been a separate PR. But it increases performance of Python completions quite a lot (orders of magnitude in some cases)

Reply:
Any reason to keep `extractParams`?

Reply:
I think `extractParams` is slightly more accurate. I believe it's still used somewhere? I'll check.

Reply:
Yeah, it's still used for getting parameter hints. `extractParamsFast` isn't good for that (but is only necessary when there are going to be a lot of function signatures to process, which isn't the case when you're doing parameter hinds)